<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ArcGate2168" failures="0" tests="6" name="sila_test_cases.CheckHandle" time="6.554" errors="0" timestamp="2020-10-07T17:17:48 IST" skipped="0">
  <testcase classname="sila_test_cases.CheckHandle" name="Test_01_check_with_available_user_handle" time="1.814"/>
  <system-out/>
  <testcase classname="sila_test_cases.CheckHandle" name="Test_02_check_with_already_taken_user_handle" time="0.488"/>
  <system-out/>
  <testcase classname="sila_test_cases.CheckHandle" name="Test_03_check_with_empty_user_handle" time="0.489"/>
  <system-out/>
  <testcase classname="sila_test_cases.CheckHandle" name="Test_04_check_with_invalid_authSignature" time="1.728"/>
  <system-out/>
  <testcase classname="sila_test_cases.CheckHandle" name="Test_05_check_with_invalid_handle_pattern" time="0.616"/>
  <system-out/>
  <testcase classname="sila_test_cases.CheckHandle" name="Test_06_check_with_invalid_App_handle" time="1.419"/>
  <system-out/>
</testsuite> <!-- sila_test_cases.CheckHandle -->
